src/helpers/pivot/pivot_time_adapter.ts

```diff
@@ -48,8 +48,8 @@ const dayAdapter: PivotTimeAdapterNotNull<number> = {
     };
   },
   toFunctionValue(normalizedValue) {
-    const date = toNumber(normalizedValue, DEFAULT_LOCALE);
-    return `"${formatValue(date, { locale: DEFAULT_LOCALE, format: "mm/dd/yyyy" })}"`;
+    const jsDate = toJsDate(normalizedValue, DEFAULT_LOCALE);
+    return `DATE(${jsDate.getFullYear()}, ${jsDate.getMonth() + 1}, ${jsDate.getDate()})`;
```

Comment:
nitpick: there's no space at all inside generated `PIVOT.VALUE` formulas ... except here. We should stay consistent.